caf-logging-serilog/.gitignore

```diff
@@ -0,0 +1,310 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
```

Comment:
If we're creating a `.gitignore` file at this level then do we really need the changes to the repository level `.gitignore` file?

Reply:
Now using vs specific only at that level, and replaced the content with source from [here](https://github.com/github/gitignore/blob/main/VisualStudio.gitignore)